rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.78.0"
+channel = "1.79.0"
```

Comment:
Could you please make this change to a separate PR?

Reply:
https://github.com/spinkube/containerd-shim-spin/pull/191